common/views/components/NewsletterPromo/NewsletterPromo.tsx

```diff
@@ -206,18 +204,11 @@ const NewsletterPromo = () => {
                       onSubmit={handleSubmit}
                       noValidate={isEnhanced}
                     >
-                      <input type="hidden" name="userid" value="225683" />
                       <input
                         type="hidden"
-                        name="SIG22a9ece3ebe9b2e10e328f234fd10b3f5686b9f4d45f628f08852417032dc990"
-                        value=""
+                        name="addressBookId"
```

Comment:
Any reason we shouldn't change addressBook to campaign here too? My take is that campaign is a more universally accepted term across newsletters. I'm not familar with dotdigital, but many other third parties use campaign.

Reply:
hmm on review of dotdigital documentation i realise campaign and addressbook are different things - ignore me

Reply:
You’re right, we should be consistent – but it turns out Campaign is a mistake I've introduced, I'll revert it.

<img width="308" alt="Screenshot 2022-07-25 at 09 50 24" src="https://user-
images.githubusercontent.com/301220/180737321-cded33fd-4c3a-40f3-a1ad-150c77694dee.png">